src/include/wiredtiger.in

```diff
@@ -7086,6 +7086,103 @@ extern int wiredtiger_extension_terminate(WT_CONNECTION *connection);
+ * Note that the verbose categories cover a wide set of sub-systems and operations
+ * within WiredTiger. As such, the categories are subject to change and evolve
+ * between different WiredTiger releases.
+ */
+typedef enum {
+/* VERBOSE ENUM START */
+    WT_VERB_API,                  /*!< API messages. */
+    WT_VERB_BACKUP,               /*!< Backup messages. */
+    WT_VERB_BLKCACHE,
+    WT_VERB_BLOCK,                /*!< Block manager messages. */
+    WT_VERB_CHECKPOINT,           /*!< Checkpoint messages. */
+    WT_VERB_CHECKPOINT_CLEANUP,
+    WT_VERB_CHECKPOINT_PROGRESS,  /*!< Checkpoint progress messages. */
+    WT_VERB_COMPACT,              /*!< Compact messages. */
+    WT_VERB_COMPACT_PROGRESS,     /*!< Compact progress messages. */
+    WT_VERB_DEFAULT,
+    WT_VERB_ERROR_RETURNS,
+    WT_VERB_EVICT,                /*!< Eviction messages. */
+    WT_VERB_EVICTSERVER,          /*!< Eviction server messages. */
+    WT_VERB_EVICT_STUCK,
+    WT_VERB_EXTENSION,            /*!< Extension messages. */
+    WT_VERB_FILEOPS,
+    WT_VERB_GENERATION,
+    WT_VERB_HANDLEOPS,
+    WT_VERB_HS,                   /*!< History store messages. */
+    WT_VERB_HS_ACTIVITY,          /*!< History store activity messages. */
+    WT_VERB_LOG,                  /*!< Log messages. */
+    WT_VERB_LSM,                  /*!< LSM messages. */
+    WT_VERB_LSM_MANAGER,
+    WT_VERB_MUTEX,
+    WT_VERB_METADATA,             /*!< Metadata messages. */
+    WT_VERB_OUT_OF_ORDER,
+    WT_VERB_OVERFLOW,
+    WT_VERB_READ,
+    WT_VERB_RECONCILE,            /*!< Reconcile messages. */
+    WT_VERB_RECOVERY,             /*!< Recovery messages. */
+    WT_VERB_RECOVERY_PROGRESS,    /*!< Recovery progress messages. */
+    WT_VERB_RTS,                  /*!< RTS messages. */
+    WT_VERB_SALVAGE,              /*!< Salvage messages. */
+    WT_VERB_SHARED_CACHE,
+    WT_VERB_SPLIT,
+    WT_VERB_TEMPORARY,
+    WT_VERB_THREAD_GROUP,
+    WT_VERB_TIERED,               /*!< Tiered messages. */
+    WT_VERB_TIMESTAMP,            /*!< Timestamp messages. */
+    WT_VERB_TRANSACTION,          /*!< Transaction messages. */
+    WT_VERB_VERIFY,               /*!< Verify messages. */
+    WT_VERB_VERSION,              /*!< Version progress messages. */
+    WT_VERB_WRITE,
+/* VERBOSE ENUM STOP */
+    WT_VERB_NUM_CATEGORIES
+} WT_VERBOSE_CATEGORY;
+/*! @} */
+
+/*******************************************
+ * Verbose levels
+ *******************************************/
+/*!
+ * @name Verbose levels
+ * @anchor verbose_levels
+ * @{
+ */
+/*!
+ * WiredTiger verbosity levels. The levels define a range of severity categories, with
+ * \c WT_VERBOSE_ERROR being the lowest, most critical level (used by messages on critical error
+ * paths) and \c WT_VERBOSE_DEBUG being the highest verbosity/informational level (mostly adopted
+ * for debugging).
+ */
+typedef enum {
+    WT_VERBOSE_ERROR = -3,  /*!< Error conditions triggered in WiredTiger. */
```

Comment:
Why is this set to -3?

Reply:
We wanted to:

- match the [MongoDB severity levels](https://github.com/mongodb/mongo/blob/master/src/mongo/logv2/log_severity.h#L54-L69).
- make some messages mandatory (as in always being logged), i.e all messages that have a severity level less than `0`
- make the calling app decide between `WT_VERBOSE_INFO` or `WT_VERBOSE_DEBUG` only.

Reply:
This will probably make more sense when the PR for SERVER-61744 will be available.

Reply:
The main point I wanted to clarify from @etiennept comment is that verbosity levels with negative values are considered unmaskable, as error/warning message usually contain important diagnostic information. 
(Roughly) similar to MongoDB, the user can configure categories with verbosity levels (via the configuration interface) with only positive values (`>= 0`, at this time, is just `WT_VERBOSE_INFO`,`WT_VERBOSE_DEBUG` (though could be extended in the future). 
As such, we wanted to split unmaskable and maskable verbosity levels between negative and positive ranges.